plugins/browser-plugin-media-tracking/src/eventGroups.ts

```diff
@@ -0,0 +1,20 @@
+import { SnowplowMediaEvent } from './snowplowEvents';
+import { EventGroup } from './types';
+import { DocumentEvent, MediaEvent, TextTrackEvent, VideoEvent } from './wgEvents';
+
+export const AllEvents: any[] = Object.values(MediaEvent);
+for (let e of Object.values(SnowplowMediaEvent)) {
+  AllEvents.push(e);
+}
+
```

Comment:
I think you can spread these together. I haven't tried this...

```
export const AllEvents: any[] = { ...Object.values(MediaEvent), ...Object.values(SnowplowMediaEvent) };
```

However... I don't think `Object.values` works in IE 😬 Might need to find a new strategy for this. Oh and no `any`s 😛